tests/cases.py

```diff
@@ -1083,6 +1084,7 @@ def test_score_r(self) -> None:
         self._test_score(
             score=self.instance.score_r,
             columns=[0, 2],
+            # switch to effective_num_relations?
```

Comment:
TODO - either switch to `self.instance.effective_num_relations` or update the tests to new have parameters

Reply:
in 3104af98 I witched it to `self.instance.effective_num_relations` and this made the tests pass again, except inductive GNN nodepiece